java/arcs/android/storage/service/IDevToolsStorageManager.aidl

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.android.storage.service;
+
+import arcs.android.storage.service.IResultCallback;
+
+/**
+ * Exposed API to manage storage for DevTools
+ */
+interface IDevToolsStorageManager {
+
+  String getStorageKeys();
```

Comment:
A javadoc style comment for this would be good to have. For example, someone might ask "which storage keys?"